messages/dev.generate.library.md

```diff
@@ -4,7 +4,9 @@ Generate a new library.
 
 # description
 
-This will clone the template repo 'forcedotcom/library-template' and update package properties
+This command is interactive. You're prompted for information to populate the new library, such as the npm scope (which must start with "@"), the name and description of the library, and its GitHub organization. The command clones the 'forcedotcom/library-template' GitHub repository, installs the plug-in's npm package dependencies using yarn install, and updates the package properties.
```

Comment:
`installs plugin-in's npm...` -> `install library's npm ...`

Reply:
Updated. Thx.